src/items/external-blocks.md

```diff
@@ -85,13 +90,16 @@ There are also some platform-specific ABI strings:
 * `extern "vectorcall"` -- The `vectorcall` ABI -- corresponds to MSVC's
   `__vectorcall` and clang's `__attribute__((vectorcall))`
 
-Finally, there are some rustc-specific ABI strings:
+<<<<<<< HEAD
```

Comment:
oops? ;)

Reply:
That never happened :P 

I need to figure out a way to rebase the changes done to my branch via the suggestions on top of local modifications without "spurious" merge commits or these issues :/